modules/redis/redis.c

```diff
@@ -346,6 +346,13 @@ redis_dd_init(LogPipe *s)
   if (!log_dest_driver_init_method(s))
     return FALSE;
 
+  if (!self->key && !self->param1)
+    {
+      msg_error("Error initializing Redis destination: command option MUST be set!",
```

Comment:
Please do not use exclamation mark here.

Reply:
Thx, fixed